audio/Android.mk

```diff
@@ -76,6 +76,10 @@ ifeq ($(strip $(TINYHAL_COMPRESS_PLAYBACK)),true)
 LOCAL_CFLAGS += -DTINYHAL_COMPRESS_PLAYBACK
 endif
 
+ifeq ($(TARGET_DEVICE),kingfisher)
```

Comment:
Please don't create defines and dependencies for a specific target. Make a generic define that describes the functionality it controls and define it in your device.mk. For example see the 3 defines above (lines 66..77)